src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -82,10 +80,7 @@ def create_user(
     logger.info("Created user with id: '{}'.", user.id)
     FidesUserPermissions.create(
         db=db,
-        data={
-            "user_id": user.id,
-            "scopes": [],
-        },
+        data={"user_id": user.id, "roles": []},
```

Comment:
Still creating a user with no roles by default but that is subject to change in https://github.com/ethyca/fides/issues/2866